web/client/themes/default/less/map-editor.less

```diff
@@ -0,0 +1,29 @@
+.map-editor-modal {
+    z-index: 2995 !important;
+    .modal-footer {
+        border-top: 1px solid #d8ddde !important;
+        .btn-group {
+            button {
+                margin-left: 2px !important;
```

Comment:
i don't like much the usage of !important.
you could not maybe find an alternative solution?

Reply:
Changed any "important" statement with a more specific selector in any possible case